.github/workflows/update_sdk.yml

```diff
@@ -0,0 +1,132 @@
+name: Update SDK
+
+on:
+  schedule:
+    - cron: "0 * * * *" # Run each hour
+  workflow_dispatch:
+
+jobs:
+  update-sdk:
+    name: "Update ${{ matrix.sdk.commit_message }}"
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        sdk:
+          - name: "Microsoft.WindowsAppSDK"
+            commit_message: "Windows App SDK"
+            allow_prerelease: false
+            script_file_name: "scripts\\get_appsdk.ps1"
+            json_file_name: "WindowsAppSDK.json"
+          - name: "Microsoft.Web.WebView2"
+            commit_message: "Microsoft.Web.WebView2"
+            allow_prerelease: false
+            script_file_name: "scripts\\get_appsdk.ps1"
+            json_file_name: "WindowsAppSDK.json"
+          - name: "Microsoft.Windows.SDK.Win32Metadata"
+            commit_message: "Win32Metadata"
+            allow_prerelease: true
+            script_file_name: "scripts\\get_win32.ps1"
+            json_file_name: "Windows.Win32.json"
+          - name: "Microsoft.Windows.SDK.Contracts"
+            commit_message: "Windows SDK"
+            allow_prerelease: true
+            script_file_name: "scripts\\get_winrt.ps1"
+            json_file_name: "WindowsSDK.json.xz"
```

Comment:
I cannot do that, see the line 108

Reply:
With "WindowsSDK.json.xz", line 108

```
7z a -txz win32more\win32generator\resources\metadata\${{ matrix.sdk.json_file_name }}.xz winmd-printer\${{ matrix.sdk.json_file_name }}
```

will be expaneded to

```
7z a -txz win32more\win32generator\resources\metadata\WindowsSDK.json.xz.xz winmd-printer\WindowsSDK.json.xz
```

I think this command line will fail because "winmd-printer\WindowsSDK.json.xz" doesn't exist.

Reply:
Oh yeah totally, it was a copy-paste error.
Fixed in https://github.com/ynkdir/py-win32more/pull/66/commits/84043875d58e79ae56d2bc59e24f91d1e8a6ecff